src/rust/generator-dispatcher/src/config.rs

```diff
@@ -0,0 +1,39 @@
+use kafka::{
+    ConfigurationError,
+    Consumer,
+};
+use rust_proto_new::SerDe;
+
+#[derive(clap::Parser, Debug)]
+pub struct GeneratorDispatcherConfig {
+    #[clap(flatten)]
+    pub kafka_config: KafkaConsumerConfig,
+}
+
+#[derive(clap::Parser, Debug)]
+pub struct KafkaConsumerConfig {
```

Comment:
The idea I had here was to eventually share this struct services - I don't currently see any services that consume from more than one consumer group, hence the somewhat generic env variable names

Reply:
Yeah this can probably go into the `kafka` library, and we should refactor s.t. the constructors take this type of config object instead of a bunch of strings. I don't personally care whether that happens now or later.